prerequirements.txt

```diff
@@ -1 +1,2 @@
 gdal==1.10.0
+protobuf==3.1.0
```

Comment:
Protobuf is a requirement of the mapbox-vector-tiles package which is a
dependency of TileStache. The issue is the newest version of protobuf
breaks backwards compatibility,  this statement forces a downgrade of
protobuf to a version that is compatible with the current pinned
version.

See: https://github.com/OpenGeoscience/ktile/issues/20 and the commit message for https://github.com/OpenGeoscience/ktile/pull/21/commits/b5583ce10925d0057973e3673ba4d0f9fe1fa735

/Chris